ndcube/ndcube.py

```diff
@@ -393,44 +390,24 @@ def axis_world_coords(self, *axes, **kwargs):
                     # other dimensions all have 0 pixel value.
                     # Replace array in quantity list corresponding to current axis with
                     # np.arange array.
-
-                    # Check for the edges arguments
-                    if edges:
-                        # Setting start and stop ranges for `pixel_edges`
-                        start = -0.5
-                        stop = cube_dimensions[axis] + 0.5
-                        quantity_list = [
-                        u.Quantity(np.zeros(cube_dimensions[dependent_axes[i]]+1),
-                                   unit=u.pix)] * n_dimensions
-                        quantity_list[axis] = u.Quantity(np.arange(start,stop), unit=u.pix)
-                    else:
                         quantity_list = [
-                        u.Quantity(np.zeros(cube_dimensions[dependent_axes[i]]),
+                        u.Quantity(np.zeros(_get_dimension_for_pixel(cube_dimensions[dependent_axes[i]], edges)),
```

Comment:
I think this should be
```u.Quantity(np.zeros(_get_dimension_for_pixel(cube_dimensions[dependent_axes[i]], edges).shape),```
since ```np.zeros``` take a shape tuple, not array values.

Reply:
Also, the indentation looks wrong.

Reply:
Ah!  Sorry.  You used  ```_get_dimension_for_pixel ```.  Forget my comment.

Reply:
Let me enclose it in a tuple, that should serve the purpose?

Reply:
No no.  I think it's fine as is.